homeassistant/components/template/template_entity.py

```
@@ -445,13 +452,13 @@ def async_start_preview(
     ) -> CALLBACK_TYPE:
         """Render a preview."""
 
-        def log_template_error(level: int, msg: str) -> None:
-            preview_callback(None, None, None, msg)
+        def suppress_preview_errors(level: int, msg: str) -> None:
+            pass
```

Add a docstring explaining why why ignore the errors